Formula/uuu.rb

```diff
@@ -0,0 +1,28 @@
+class Uuu < Formula
+  desc "Universal Update Utility, mfgtools 3.0. NXP I.MX Chip image deploy tools"
+  homepage "https://github.com/NXPmicro/mfgtools"
+  url "https://github.com/NXPmicro/mfgtools/releases/download/uuu_1.4.139/uuu_source-1.4.139.tar.gz"
+  sha256 "25a24575f9e4781b78f8f0930f751ed371dc1a00fc22d543e9cbe4a9a70a615c"
+  license "BSD-3-Clause"
+
+  head "https://github.com/NXPmicro/mfgtools.git", branch: "master"
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+
+  # Change to stable when libusb 1.0.25 is released
+  # https://github.com/libusb/libusb/pull/911
+  depends_on "libusb" => :head
```

Comment:
I don't think this works.

Reply:
@SMillerDev, is is okay if I check for version of libusb in the `.install` block and output a warning to user? Are there any ways to output warnings nicely?

Reply:
We don't want to warn users on install if we can just remove the dependency all together.

Reply:
@SMillerDev, it is runtime dependency, uuu tool won't work without it.

Reply:
The `install` method is never run by the vast majority of users, so there's not much point in putting warnings there. Those will just get lost in CI logs.

Reply:
If it's a runtime dependency for the current version, do 

```suggestion
  depends_on "libusb"
```
If it is not, do
```suggestion
```